AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -243,13 +248,18 @@ - (void)createWithPartition:(NSString *)partition
                                                                  }
                                                                  MSLogDebug([MSDataStore logTag], @"Document json:%@", json);
 
+                                                                 // Create an instance of Document.
+                                                                 // TODO fix the warning.
```

Comment:
What warning are you getting here?

Reply:
it's benign warning: Messaging unqualified id